src/style.css

```diff
@@ -1,3 +1,74 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 100%;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.nav {
+  margin: 0;
+  padding: 0;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  align-items: center;
+}
+
+.nav__item {
+  margin-left: 20px;
+  text-transform: uppercase;
+  list-style: none;
+}
+
+.nav__item:nth-child(1) {
+  margin-left: 0;
+}
+
+a {
+  display: block;
+  align-items: center;
```

Comment:
The 'align-items: center;' property is used on an 'a' tag, which is not a flex container. This property will have no effect here. Consider removing it or applying it to a flex container.